x-pack/filebeat/module/crowdstrike/falcon/config/falcon.yml

```diff
@@ -0,0 +1,19 @@
+type: log
```

Comment:
I wonder if it's worth also supporting the syslog input here? https://www.crowdstrike.com/blog/tech-center/integrate-with-your-siem/ seems to suggest that it's possible for the Connector to send date directly to a syslog server?

Reply:
It would be easy to add the config option for syslog and maybe not make it the default choice. But my concern is we can't test it to validate that the output is identical to the file output. I'm on the fence but can be convinced to add it if it's not the default choice.

Reply:
I see, makes sense. I think we can leave it out and add it if it's being requested. Then we have a way to validate it.